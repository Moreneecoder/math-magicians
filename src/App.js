function App() {
  return (
    <div className="App">
      <h1>Math-Magician SetUp Works Fine</h1>
    </div>
  );
}

export default App;
