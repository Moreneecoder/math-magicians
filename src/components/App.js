import Display from './Display';
import ButtonPanel from './ButtonPanel';

function App() {
  return (
    <div className="App">
      <>
        <Display result="23" />
        <ButtonPanel />
      </>
    </div>
  );
}

export default App;
