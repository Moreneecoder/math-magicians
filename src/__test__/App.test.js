import renderer from 'react-test-renderer';
import App from '../components/App';

describe('App Component', () => {
  test('Renders App component as expected', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
