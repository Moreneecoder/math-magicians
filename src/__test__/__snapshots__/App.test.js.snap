// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Renders App component as expected 1`] = `
<div
  className="App"
>
  <div
    className="display"
  >
    <h1>
      0
    </h1>
  </div>
  <div
    className="ButtonPanel"
  >
    <div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          %
        </button>
      </div>
      <div
        className="Button defaultBg"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
      </div>
    </div>
    <div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          7
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          8
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          9
        </button>
      </div>
      <div
        className="Button defaultBg"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          X
        </button>
      </div>
    </div>
    <div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          4
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          5
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          6
        </button>
      </div>
      <div
        className="Button defaultBg"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          -
        </button>
      </div>
    </div>
    <div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          1
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          2
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          3
        </button>
      </div>
      <div
        className="Button defaultBg"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
    </div>
    <div>
      <div
        className="zeroButton "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          0
        </button>
      </div>
      <div
        className="Button "
      >
        <button
          onClick={[Function]}
          type="button"
        >
          .
        </button>
      </div>
      <div
        className="Button defaultBg"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
